lib/GraphApplication.php

```diff
@@ -11,6 +11,8 @@ final class GraphApplication
      * @throws \Safe\Exceptions\JsonException
      *
      * @throws \Safe\Exceptions\FilesystemException
+     *
+     * @api
```

Comment:
work arround until https://github.com/TomasVotruba/unused-public/pull/79 is merged/released